src/Tribe/Tickets.php

```diff
@@ -330,7 +340,16 @@ public function get_tickets_ids( $post = null ) {
 				$args = $this->get_tickets_query_args();
 			}
 
+			$cache = new Tribe__Cache();
+			$cache_key = $cache->make_key( $args );
+			$query = $cache->get( $cache_key );
+
+			if ( $query instanceof WP_Query ) {
+				return $query->posts;
+			}
+
 			$query = new WP_Query( $args );
+			$cache->set( $cache_key, $query, $cache::NO_EXPIRATION );
```

Comment:
99% sure referencing a class constant via the object reference works only with PHP 5.3+. For compatibility with 5.2, we probably need `Tribe__Cache::NO_EXPIRATION`.

Reply:
Yeah @barryhughes, `constant( $cache . '::NO_EXPIRATION' )` is what we want.

Reply:
🥇